executable/te.ksy

```diff
@@ -0,0 +1,125 @@
+meta:
+  id: te
+  title: TE (Terse Executable) file format. A stripped version of PE (Portable Executable) format.
```

Comment:
"... format" and "Stripped version of ..." should not part of the title. The latter should be part of top-level `doc`.